geomeppy/idf.py

```diff
@@ -14,6 +14,7 @@
 from .patches import PatchedIDF
 from .recipes import set_default_constructions, set_wwr, rotate, scale, translate, translate_to_origin
 from .view_geometry import view_idf
+from .utilities import core_perim_zone_coordinates
```

Comment:
Maybe move this to a module called `core_perim`, then you can do something like:

    import core_perim
    coords = core_perim.zone_coordinates(block.coordinates, block.perim_depth)